### DISCUSSION TITLE
Docs website is down

### DISCUSSION THREAD

**Discussion by jgburford** [idom-team/idom#465] [Problem] [answered]
The docs site https://idom-docs.herokuapp.com/ is giving an "application error" message.

Apologies if this isn't the right place to report this.

**Comment by rmorshea** [comment, marked as answer]
I'll address this shortly. Thanks for reporting, I'll be tracking this [here](https://github.com/idom-team/idom/issues/466).

And yeah, this is the right place to report things though. I like to keep issues creation to a minimum if I can.

**Reply by jgburford** [reply]
Thank you!

Just an FYI, and not sure whether it's only on my system, but Firefox doesn't finish loading the documentation pages. Anytime it runs into the <script> tags that are loading up the live examples, it seems to render nothing after that.

Chrome-based browsers appear unaffected by this issue.

**Reply by rmorshea** [reply]
Tracking that [here](https://github.com/idom-team/idom/issues/469)

**Reply by rmorshea** [reply]
Firefox should be working now.

**Comment by rmorshea** [comment]
In terms of where to report things, I've actually just changed things. There's now a Bug Report form for reporting issues, and the "Problem" discussion section has been removed. So bugs and problems can be reported via the "Issues" tab and questions should be posted under the "Discussions" tab.
